netshare/generate/model_generate.py

```diff
@@ -112,44 +119,28 @@ def merge_attr(config_group: dict, configs: List[dict]) -> None:
             os.path.join(attr_raw_npz_folder, "chunk_id-{}.npz".format(chunkid))
         )["data_attribute"]
 
-        if num_chunks > 1 and word2vec_size:
```

Comment:
Whether we need to concat and merge the attributes is not based on if the user specifies `word2vec_size`. I just deleted that condition here.